### PR TITLE
[docs] Add YBDB feature intro

### PR DESCRIPTION
**Pull request by ddhodge**
Add page about YBDB features as intro/preamble to Explore

@netlify /

### CODE REVIEW

**Comment by aravind-nallan-yb** [comment]
LGTM

**Comment by ddhodge** [comment]
@aravind-nallan-yb @aishwarya24 I haven't decided whther this is a suitable spot, or if it should rather go in a first topic in Explore - with links from each section to a relevant Explore section, for example.

File: docs/content/preview/features.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: Benefits of using YugabyteDB
+headerTitle: Key benefits
+linkTitle: Key benefits
+description: Benefits of using YugabyteDB to power your next cloud-native application.
+headcontent: Why YugabyteDB should power your next cloud-native application
+type: docs
+---
+
+YugabyteDB is a high-performance distributed SQL database for powering global, internet-scale applications. Built using a combination of high-performance document store, per-shard distributed consensus replication, and multi-shard ACID transactions (inspired by Google Spanner), YugabyteDB serves both scale-out RDBMS and internet-scale OLTP workloads with low query latency, extreme resilience against failures, and global data distribution. As a cloud-native database, it can be deployed across public and private clouds as well as in Kubernetes environments.
```
Comment (premkumr) [inline docs/content/preview/features.md:10]: Could we add this on the "Documentation Home" page too, just before the "Get started in the Cloud" box ?

**Review by premkumr — APPROVED** [review]
LGTM!!!

File: docs/content/_index.md [hunk 2]
```diff
@@ -37,46 +37,46 @@ resources:
...
   >}}
 
   {{< sections/3-box-card
-    title="Run a real world demo app"
-    description="Run a distributed full-stack e-commerce application built on YugabyteDB, Node.js Express, and React."
-    buttonText="Get started"
-    buttonUrl="/preview/develop/realworld-apps/ecommerce-app/"
-    imageAlt="Run a real world demo app"
-    imageUrl="/images/homepage/run-real-world-demo-app.svg"
+    title="For developers"
```
Comment (premkumr) [inline docs/content/_index.md:53]: should it not be "For **D**evelopers"?   as we have "Get **S**tarted" ..

**Review by aishwarya24 — APPROVED** [review]
Looks great!

**Pull request merged**